service/controller/app/resource/chart/resource.go

```diff
@@ -108,42 +107,35 @@ func (r *Resource) removeFinalizer(ctx context.Context, chart *v1alpha1.Chart) e
 	return nil
 }
 
-// equals asseses the equality of ReleaseStates with regards to distinguishing fields.
-func equals(current, desired *v1alpha1.Chart) bool {
-	if current.Name != desired.Name {
-		return false
-	}
-	if !reflect.DeepEqual(current.Spec, desired.Spec) {
-		return false
-	}
-	if !reflect.DeepEqual(current.Labels, desired.Labels) {
-		return false
+func copyChart(current *v1alpha1.Chart) *v1alpha1.Chart {
```

Comment:
`resourceVersion`, `UID`, and many other variables hinder to see the real difference between the desired and current status of object. So I make a copy to the `currentObject` by using this `copyChart`.

Reply:
It makes sense to filter it but I'm not sure on the name.

But I can't think of a better name either. With some comments I think it will be fine. See next comment.